packages/teams-js/src/internal/handlers.ts

```diff
@@ -19,17 +19,17 @@ class HandlersPrivate {
   public static handlers: {
     [func: string]: Function;
   } = {};
-  public static themeChangeHandler: (theme: string) => void;
+  public static themeChangeHandler: null | ((theme: string) => void);
```

Comment:
>themeChangeHandler [](http://example.com/codeflow?start=16&length=18)

All of these handlers are eventually set to `null` in the `uninitializeHandlers()` method, and this reflects that reality.